apps/endatix-hub/features/public-form/ui/survey-component.tsx

```diff
@@ -58,7 +60,57 @@ export default function SurveyComponent({ definition, formId, submission }: Surv
     });
   }, []);
 
+  const interceptImages = useCallback((sender: SurveyModel, options: ValueChangingEvent) => {
+    const resizeImage = (imgSrc: string, type: string): Promise<string> => {
+      return new Promise((resolve) => {
+        const img = new Image();
+        img.onload = () => {
+          let width = img.width;
+          let height = img.height;
+    
+          if (width > height) {
```

Comment:
I want to recommend a simplification to make this more expressive and easier to understand. What do you think of this:

```ts
if (width > height && width > MAX_IMAGE_SIZE) {
  // handle this case
} else if (height > MAX_IMAGE_SIZE) {
 // handle this case
}
```

Reply:
Done